configure

```diff
@@ -1989,6 +1993,25 @@ Available commands are:
       message.index("\n") != nil
     end
   end
+
+  # Returns true if MacPorts is installed in its default location.
+  def macports?
+    File.exists? '/opt/local/bin/port'
```

Comment:
@YorickPeterse That's fine; I'll switch it to just checking the PATH for the *port* command. The current code for Homebrew doesn't even do that; it just calls *brew* and hopes for the best.

I'll post a patch shortly to make this change.